sdk/storage/storage-queue/src/generated/src/models/index.ts

```diff
@@ -514,13 +514,6 @@ export interface MessageIdDeleteExceptionHeaders {
   clientRequestId?: string;
 }
 
-/** Known values of {@link GeoReplicationStatusType} that the service accepts. */
-export const enum KnownGeoReplicationStatusType {
```

Comment:
Sounds like a breaking change, please document it into ChangeLog.md

Reply:
It's not a breaking comparing to the last release. We've added 'KnownGeoReplicationStatusType ' before, and this is to revert it, because it's an interface change, we'll need to get arch board team approval to merge it.